pkg/collector/corechecks/servicediscovery/usm/erlang.go

```diff
@@ -0,0 +1,82 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Would be nice to have prefixes in the titles of PR in case they are only touching one component.